scraping/scraper.py

```diff
@@ -67,16 +67,20 @@ def scrape_cluster(workspace, cluster_dict, instance_types, session, api, result
         state_message=cluster_dict["state_message"],
         driver_type=cluster_dict["driver_node_type_id"],
         worker_type=cluster_dict["node_type_id"],
-        num_workers=cluster_dict["num_workers"],
+        num_workers=cluster_dict.get("num_workers", 0),
+        autoscale_min_workers=(cluster_dict
+                               .get('autoscale', {})
+                               .get('min_workers', 0)),
+        autoscale_max_workers=(cluster_dict
+                               .get('autoscale', {})
+                               .get('max_workers', 0)),
         spark_version=cluster_dict["spark_version"],
-        creator_user_name=cluster_dict["creator_user_name"],
+        creator_user_name=cluster_dict.get("creator_user_name", "DELETED"),
```

Comment:
What happens, if:
1. the cluster is created, we scrape the user who exists at this moment
2. the user is deleted
3. We scrape again ?

The reason I want to clarify this is that if we overwrite the user, we lose the information about the creator, even though we have the information.

Reply:
Good point. I'm going to change so if we already have a user we wont change it.